src/App.tsx

```diff
@@ -1,9 +1,36 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    const list = document
+      .querySelector('.wrapper')?.children[0].children;
+
+    if (list !== undefined) {
+      for (let i = 0; i < list.length; i += 1) {
+        list[i].classList.add('animate__animated');
+        list[i].classList.add('animate__flash');
+      }
+
+      setTimeout(() => {
+        for (let i = 0; i < list.length; i += 1) {
+          list[i].classList.remove('animate__animated');
+          list[i].classList.remove('animate__flash');
+        }
+      }, 500);
+    }
```

Comment:
Let's do it with state like
`const [isLoadiing, setIsLoading] = useState(true)`
and use classname for passing these classes


In setTimeout you just set loading to be `false`